build.gradle

```diff
@@ -88,6 +88,11 @@ subprojects { subproject ->
     }
 
 	configurations {
+		all {
+			resolutionStrategy.dependencySubstitution {
+				substitute module('org.hamcrest:hamcrest-core') using module('org.hamcrest:hamcrest:2.2') because 'junit depends on hancrest-core, but a ll hamcrest-core classes have been incorporated into hamcrest'
```

Comment:
```suggestion
				substitute module('org.hamcrest:hamcrest-core') using module('org.hamcrest:hamcrest:2.2') because 'junit depends on hamcrest-core, but all hamcrest-core classes have been incorporated into hamcrest'
```